common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js

```diff
@@ -230,10 +232,51 @@ const IIIFPresentationDisplay = ({
   const [viewType, setViewType] = useState<ViewType>('unknown');
   const [imageThumbnails, setImageThumbnails] = useState([]);
   const [imageTotal, setImageTotal] = useState(0);
+  const [secondsPlayed, setSecondsPlayed] = useState(0);
+  const [isPlaying, setIsPlaying] = useState(false);
   const iiifPresentationManifest = useContext(ManifestContext);
   const video = getVideo(iiifPresentationManifest);
   const audio = getAudio(iiifPresentationManifest);
 
+  function trackViewingTime() {
+    trackEvent({
+      category: 'Engagement',
```

Comment:
Have we invented this?

Do we have a list of them somewhere? If not, could be good, and potentially have definitions as to what they are to help with choosing another in the future?